src/lib/kdump/kdump_calibrator.rb

```diff
@@ -0,0 +1,159 @@
+module Yast
+  # This class tries to calibrate Kdump minimum and maximum values
+  #
+  # It relies on kdumptool but, if the tool is not available or does not work
+  # as expected, tries to set up reasonable memory values on its own.
+  class KdumpCalibrator
+    include Yast::Logger
+
+    LOW_MEM = 896
+    MIN_LOW_DEFAULT = 72
+    MB_SIZE = 1048576
+
+    KDUMPTOOL_CMD = "kdumptool %s calibrate"
+    KDUMPTOOL_ARG = "--configfile '%s'"
```

Comment:
for constants with replacement I prefer to use verbose variant like `%{configpath}`